app/api/src/endpoints/v1/layer_library.py

```diff
@@ -133,17 +128,12 @@ async def update_style(
     return style
 
 
-@styles_router.delete("/{name}", response_model=models.StyleLibrary)
-async def delete_a_style_library(
-    name: str,
+@styles_router.delete("")
+async def delete_style_libraries(
+    name: List[str] = Query(default=None),
     db: AsyncSession = Depends(deps.get_db),
     current_user: models.User = Depends(deps.get_current_active_superuser),
 ):
-    style = await crud.style_library.get_by_key(db, key="name", value=name)
-    if not style:
-        raise HTTPException(status_code=404, detail="style library not found.")
-    else:
-        style = style[0]
 
-    style = await crud.style_library.remove(db, id=style.id)
-    return style
+    await crud.style_library.remove_multi_by_key(db, key="name", values=name)
+    return "ok"
```

Comment:
Concerning the return message. Maybe it is also a global thing but is returning just 'ok' enough? Or should we add something like a dict.

Reply:
to document:
The client mostly understands the success by status code (As mostly it is checked by some packages. And when it's needed the content is checked.)
as [RFC](https://www.rfc-editor.org/rfc/rfc7231#section-4.3.5) suggests, we can provide some details about the process. I think it would be good to show which ids was affected. However maybe it is not needed. Also we may remove the response "ok" at the moment. _Empty is better than non standard, as it would affect the future development_)

But at delete scenarios, we filter the delete by logged in user. So it is possible not to delete the item at all.
For example a situation that the user has two tabs at the same time and in the other tap, signs out and signs in again with other account. Now if they call the delete, the request returns 200, and the client side maybe removes the items, But they really have not removed. However this situation doesn't harm.

Reply:
Sounds like a good compromise now. I believe it might be a global thing we might have to standardize in the future also for the other endpoints and error logs. But for now we are good!